x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/details/execution_log_table/execution_log_table.tsx

```diff
@@ -223,7 +223,7 @@ const ExecutionLogTableComponent: React.FC<ExecutionLogTableProps> = ({
             icon: 'filter',
             type: 'icon',
             onClick: (value: object) => {
```

Comment:
You should type this eventually other than object and remove the ugly `get` below as that is just going to break later down the road.

Reply:
Yeah, that's no bueno -- will try to `type` out the columns and then this can get cleaned up 👍

Reply:
Already had a type for this! 🙂 Addressed here: https://github.com/elastic/kibana/pull/129003/files#diff-81a89a66909bd6b3a6836daa93e7cef8100675de09e94244319ee60426a4c361R230-R232